tests/conftest.py

```diff
@@ -95,7 +105,7 @@ def app(request):
     with app.app_context():
         db.create_all()
         list(search.create(ignore=[400]))
-        sleep(5)
+        sleep(10)
```

Comment:
do we really need to slow down all tests? Can we create special fixture that creates Elastic indices only for tests that really need them.